2_adventures/001/lawrencetran1/03_rubies_on_the_wall/21_days_of_ruby_on_the_wall.rb

```diff
@@ -0,0 +1,20 @@
+require 'date'
+today = Date.today
+start_of_challenge = Date.parse("April 13, 2015")
+days_done = (today - start_of_challenge).to_i
+format = "%a %m/%d"
+
+days_done.times do |i|
+  start_day = start_of_challenge + i
+  end_day = start_day + 1
+
+  puts [
+    "*",
+    sprintf("Day %02d:", i + 1),
+    start_day.strftime(format),
+    "=> " + (21-i).to_s + " days of Ruby on the wall.",
```

Comment:
Nice use of an array to present a collection of strings.  I do this pretty commonly rather than a HereDoc.

A string like line 15 can be:

    "=> #{21 - i} days of Ruby on the wall."

And could save on strings that are created and then garbage collected.  And some typing.